doc/concepts.rst

```diff
@@ -1110,8 +1110,8 @@ In the rest of this section, ``<prefix>`` refers to the directory in
 which the user chooses to install packages. When installing via Opam,
 it's Opam that sets this directory. When calling ``dune install``,
```

Comment:
```suggestion
which the user chooses to install packages. When installing via opam,
it's opam that sets this directory. When calling ``dune install``,
```

While under the rules of English language opam should be capitalised, Thomas feels strongly about it being all lowercase even when talking about the name of the package manager itself. `opam` commands will still be in monospace.

Reply:
I know! In the recent work I did on the Dune files, I capitalised them as one does with proper names, but Thomas made his preference known a few weeks ago. I'll make one PR to fix them all. 

(I don't know how to add a check in run.t, so is that something I should do? I can learn how.)